lib/screens/play/player_hand.dart

```diff
@@ -11,27 +11,41 @@ class PlayerHand extends StatelessWidget {
   PlayerHand(this.hand);
 
   double _getCardXAlignment(int index) {
-    return (2 / (max(hand.cards.length, maxCardsBeforeOverlapping) + 1)) * (index - ((hand.cards.length - 1) / 2));
+    return (2 / (max(hand.cards.length, maxCardsBeforeOverlapping) + 1)) *
+        (index - ((hand.cards.length - 1) / 2));
   }
 
   @override
   Widget build(BuildContext context) {
-    List<Widget> resourceCards = hand.cards.asMap().map((i, card) {
-      
-      return MapEntry(i, 
-        Align(
-          alignment: Alignment(_getCardXAlignment(i), _getCardXAlignment(i).abs()),
-          child: SizedBox(
-            width: (MediaQuery.of(context).size.width) / (maxCardsBeforeOverlapping + 2),
-            height: ((MediaQuery.of(context).size.width) / (maxCardsBeforeOverlapping + 2)) * 1.4,
-            child: ResourceCard(card)
-          ),
-        )
-      );
-    }).values.toList();
-
-    return Stack(
-      children: resourceCards
-    );
+    List<Widget> resourceCards = hand.cards
+        .asMap()
+        .map((i, card) {
+          return MapEntry(
+              i,
+              Align(
+                alignment: Alignment(
+                    _getCardXAlignment(i), _getCardXAlignment(i).abs() - .5),
+                child: SizedBox(
+                    width: (MediaQuery.of(context).size.width) /
+                        (maxCardsBeforeOverlapping + 2),
+                    height: ((MediaQuery.of(context).size.width) /
+                            (maxCardsBeforeOverlapping + 2)) *
+                        1.4,
+                    child: ResourceCard(card)),
+              ));
+        })
+        .values
+        .toList();
+
+    Widget _resourceCount() {
+      return Align(
+          alignment: AlignmentDirectional.topEnd, // Top right corner
+          child: Text("Count: ${hand.cards.length}"));
+    }
```

Comment:
Doesn't really make sense for this to be a nested method?
```suggestion
    final resourceCount = Align(
          alignment: AlignmentDirectional.topEnd, // Top right corner
          child: Text("Count: ${hand.cards.length}"));
    }
```